salesagility/SuiteCRM
# Fixed #2119 - Renamed AOW_Workflow.svg to AOW_WorkFlow.svg

In relation to issue #2119 

## Description



Icon renamed icon for workflow.

## Motivation and Context

This icon appears missing in the recently viewed links

## How To Test This


1. Create a workflow
2. Save

Passed: when you can see the workflow icon next to your work flow in the sidebar


## Types of changes


### Final checklist
